index.html

```diff
@@ -263,214 +272,216 @@
+                <li class="flex-center">
+                  <a href="https://dhall-lang.org/">Dhall configuration language</a>
+                </li>
+                <li class="flex-center">
+                  <a href="https://github.com/rpearce/bashcards">bashcards: CLI tool to practice flashcards</a>
+                </li>
+              </ul>
 
-              <h2 class="unit-1">AsciiDoc</h2>
-              <pre class="unit-1">image:https://builtwithnix.org/badge.svg["built with nix", link="https://builtwithnix.org/"]</pre>
+              <div class="unit-1 image">
+                <img alt="Built with Nix badge" src="img/built_with_nix.svg"/>
+              </div>
 
-              <p class="unit-1 fork">
-                Fork <a href="https://github.com/nix-community/builtwithnix.org">https://github.com/nix-community/builtwithnix.org</a><br/>
-                and submit a pull-request to list your project.
+              <p class="unit-1">
+                <strong>Is your project missing from the list?</strong><br/>
+                Share the love by embedding the badge into your project's README:
               </p>
+
+              <div class="unit-1">
+                <h2 class="unit-1">Markdown</h2>
+                <pre class="unit-1">[![built with nix](https://builtwithnix.org/badge.svg)](https://builtwithnix.org)</pre>
+
+                <h2 class="unit-1">reStructuredText</h2>
+                <pre class="unit-1">.. image:: https://builtwithnix.org/badge.svg :alt: built with nix :target: https://builtwithnix.org</pre>
+
+                <h2 class="unit-1">AsciiDoc</h2>
+                <pre class="unit-1">image:https://builtwithnix.org/badge.svg["built with nix", link="https://builtwithnix.org/"]</pre>
+
+                <p class="unit-1 fork">
+                  Fork <a href="https://github.com/nix-community/builtwithnix.org">https://github.com/nix-community/builtwithnix.org</a><br/>
+                  and submit a pull-request to list your project.
+                </p>
+              </div>
             </div>
           </div>
-        </article>
-      </div>
-    </div>
-
-    <div class="flex-center hide-on-mobile" id="icons">
-      <div class="container-fluid">
-        <div class="icons">
-          <img src="img/works_simple.svg"/>
-          <img src="img/languages_simple.svg"/>
-          <img src="img/zero_to_cloud_simple.svg"/>
-          <img src="img/packages_simple.svg"/>
-          <img src="img/compile_simple.svg"/>
+        </div>
+      </section>
+
+      <div class="flex-center hide-on-mobile" id="icons">
+        <div class="container-fluid">
+          <div class="icons">
+            <img alt="" role="img" src="img/works_simple.svg"/>
+            <img alt="" role="img" src="img/languages_simple.svg"/>
+            <img alt="" role="img" src="img/zero_to_cloud_simple.svg"/>
+            <img alt="" role="img" src="img/packages_simple.svg"/>
+            <img alt="" role="img" src="img/compile_simple.svg"/>
```

Comment:
Is the alt tag supposed to contain something?

Reply:
From above:

> We always need an alt attribute on <img /> tags. If it's decorative, specify alt="" to tell the screen reader that it's decorative. Read this to learn more: https://www.w3.org/WAI/tutorials/images/decorative/

Good question! No, for these are decorative images that – as far as I can tell – don't add content to the page.